src/routes/game/components/GameOverDialog.vue

```diff
@@ -52,35 +52,60 @@
           </div>
         </div>
       </template>
+      <p v-if="opponentWantsToRematch">
+        {{ gameStore.opponent.username }} wants to rematch
+      </p>
+      <p v-if="opponentDeclinedRematch">
+        {{ gameStore.opponent.username }} declined rematch and left the game
+      </p>
     </template>
 
     <template #actions>
-      <v-btn
-        color="surface-1"
-        variant="flat"
-        data-cy="gameover-go-home"
-        :loading="leavingGame"
-        @click="goHome"
-      >
-        Go Home
-      </v-btn>
+      <div class="d-flex gap-2">
+        <v-btn
+          class="mr-4"
+          color="surface-1"
+          variant="flat"
+          data-cy="gameover-go-home"
+          :loading="leavingGame"
+          @click="goHome"
+        >
+          Go Home
+        </v-btn>
+        <v-btn
+          color="primary"
```

Comment:
I think we want to disable this button when the opponent has declined.
```suggestion
          :disabled="opponentDeclinedRematch("
          color="primary"
```